tests/Oryx.Integration.Tests/Node/NodeTestWithAppInsightsConfigured.cs

```diff
@@ -72,26 +77,31 @@ public async Task CanBuildAndRun_NodeApp_WithAppInsights_Configured(string nodeV
         }
 
         [Theory]
+        [InlineData("8")]
+        [InlineData("9")]
         [InlineData("10")]
         [InlineData("12")]
-        public async Task CanBuildAndRun_NodeApp_WithAppInsights_Configured_By_Oryx(string nodeVersion)
+        // From 1.7.2 onward appinsights sdk have new environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING"
+        // instead  of "APPINSIGHTS_INSTRUMENTATIONKEY"
+        public async Task CanBuildAndRun_NodeApp_WithAppInsights_New_Env_Variable(string nodeVersion)
         {
             // Arrange
             var appName = "linxnodeexpress-appinsights";
             var volume = CreateAppVolume(appName);
             var appDir = volume.ContainerDir;
             var spcifyNodeVersionCommand = "--platform nodejs --platform-version=" + nodeVersion;
-            var aIKey = ExtVarNames.UserAppInsightsKeyEnv;
+            var connectionString = ExtVarNames.UserAppInsightsConnectionStringEnv;
             var aIEnabled = ExtVarNames.UserAppInsightsEnableEnv;
```

Comment:
These names are getting confusing. I would change this to have the string `Agent` in it. ex: `UserAppInsightsAgentEnv` or `UserAppInsightsAgentExtensionEnv`